packages/fxa-auth-server/lib/routes/utils/third-party-events.ts

```diff
@@ -301,20 +360,26 @@ export async function getGooglePublicKey(
  * @param clientIds
  * @param publicKeyPem
  * @param issuer
+ * @param statsd
  * @returns {Promise}
  */
 export async function validateSecurityToken(
   token: string,
   clientIds: [string],
   publicKeyPem: any,
-  issuer: string
+  issuer: string,
+  statsd: StatsD
 ) {
-  // Decode the token, validating its signature, audience, and issuer
-  return jwt.verify(token, publicKeyPem, {
-    algorithms: ['RS256'],
-    audience: clientIds,
-    issuer: issuer,
-  });
+  try {
+    return jwt.verify(token, publicKeyPem, {
+      algorithms: ['RS256'],
+      issuer,
+      audience: clientIds,
+    });
+  } catch (error) {
+    statsd.increment('validateSecurityToken.error');
+    throw new Error(`Security token validation failed: ${error.message}`);
```

Comment:
Not a big deal, but curious why we are wrapping this error message and not just re throwing the error.